lib/cache/stores/redis-cluster.js

```diff
@@ -0,0 +1,57 @@
+const _         = require('lodash');
```

Comment:
@derrickmehaffy You've worked on Redis Cluster support previously haven't you ? Do you think this new store is warranted ? Or can the existing redis store be used ?

Reply:
I'll have to do some testing, I did briefly mess with clusters and the main issue was the key partitioning, sentinels work fine.